temporaltest/tls_test.go

```diff
@@ -0,0 +1,318 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
```

Comment:
I think we can delete this test now that the PR isn't specifically adding TLS options. See other comment on adding a unit test to the liteconfig package though.